scripts/db_migrate.sh

```diff
@@ -8,14 +8,15 @@ DB_PORT=''
 DB_HOST='127.0.0.1'
 DB_USER='root'
 DB_PASS='root'
-DB_EXPORT_FLAGS='--routines --column-statistics=0 --quick --hex-blob --single-transaction'
-DB_IMPORT_FLAGS=''
+DB_EXPORT_FLAGS='--routines --quick --hex-blob --single-transaction'
 
 # Determine DB
 echo "Welcome to the SDIT DB Migrator (mysql/postgres) $NEWLINE"
 read -rp "Select (mysql|postgres):" DB_MODE
+read -rp "Select (FROM) docker container name: " CONTAINER_NAME_FROM
```

Comment:
I don't think we need these asks. It just makes this command harder to use. If you take a look at `docker ps`

```
5db86a7f4510   mariadb:10.3                    "docker-entrypoint.s…"   2 weeks ago    Up 29 hours   0.0.0.0:33103->3306/tcp   sourcetoad_mariadb103
5bae06984f18   mariadb:10.4                    "docker-entrypoint.s…"   2 weeks ago    Up 29 hours   0.0.0.0:33104->3306/tcp   sourcetoad_mariadb104
6eabb950ac3f   mariadb:10.2                    "docker-entrypoint.s…"   2 weeks ago    Up 29 hours   0.0.0.0:33102->3306/tcp   sourcetoad_mariadb102
81c4e38c7e23   jwilder/nginx-proxy             "/app/docker-entrypo…"   2 weeks ago    Up 29 hours   0.0.0.0:80->80/tcp        sourcetoad_nginx_proxy
234b917d34d5   mariadb:10.5                    "docker-entrypoint.s…"   2 weeks ago    Up 29 hours   0.0.0.0:33105->3306/tcp   sourcetoad_mariadb105
```

We might need a bit of bash, but we know if you give us 33102 we know its maria102, etc. I think if we get to ask about to 2 less things and do some guesses behind the scene. This cuts down the parameters needed to use this.